tests/QueryTest.php

```diff
@@ -7,12 +7,12 @@
 use atk4\core\AtkPhpunit;
 use atk4\dsql\Exception;
 use atk4\dsql\Expression;
+use atk4\dsql\MySQL\Query as Query_MySQL;
```

Comment:
`use atk4\dsql\Mysql` shoud be preffered instead.

also fix RandomTest (and maybe other) to use ...\tests NS.

Reply:
What do you mean with `use atk4\dsql\Mysql`?

Reply:
later refer to this import like `MySql\Connection` :) (never use aliases with _ if you can import the parent NS)

Reply:
Beautiful 😄